doc/underpinnings.tex

```diff
@@ -0,0 +1,779 @@
+joiners.  Note: this time we don't divide through by the academic
+years.  Armed with these proportions we can calculate the expectation
+as a percentage for each e.g $10\%$ to $(s_x,n_y,ay_z)$ and $90\%$ to
+$(s_j,n_k,ay_l)$.
+
+Again, rather then a direct calculation of the population to be
+assigned to each entity we introduce variance by using the multinomial
+distribution.  It performs exactly the same job as the binomial except
+over multiple possible outcomes.
+
+In a now familiar pattern, we also wish to introduce variance in the
+actual percentages used to calculate the assignment.  Like before, we
+want high confidence in values we have lots of observed data for and
+low confidence for few observations.  The Dirichlet distribution does
+this for multivariates in exactly the same way beta distribution does
+for two.  We combine the two to produce another hierarchical model.
+
+\begin{equation*}
+\begin{split}
+  \text{Multinomial} ( & j_{ay_{z}}, \\
+  & \text{Dirichlet} (\alpha^{j}_{s_x,n_y,ay_z}, \dots,
+  \alpha^{j''}_{s_{x''},n_{y''},ay_{z''}}))
+\end{split}
+\end{equation*}
+
+Here the $\alpha^{j}$s represent the total number of transitions from
+$non\-send$ to ${s_x,n_y,ay_z}$.
+
+The results of the multinomial function is however a vector of the
+counts of the joiners for each of the possible entities.  Thus for any
+particular entity of interest we need to take the appropriate index.
+
+\begin{equation*}
+  \begin{split}
+    j_{s_x,n_y,ay_z} = \bigg[\text{Multinomial}( & j_{ay_z}, \\
+    & \text{Dirichlet}(\alpha^{j}_{s_x,n_y,ay_z},\alpha^{j'}_{s_{x'},n_{y'},ay_{z'}}, \dots, \alpha^{j''}_{s_{x''},n_{y''},ay_{z''}}))\bigg]_{s_x,n_y,ay_z}
+  \end{split}
+\end{equation*}
+
+We can write this out in full for single entity by expanding
+$j_{ay_z}$.
+
+\begin{equation*}
+  \begin{split}
+j_{s_x,n_y,ay_z} = \bigg[\text{Multinomial}( & \text{Binomial}(p_{ay_z},
+\text{Beta}(\frac{\alpha^j_{ay_z}}{cy_{obs}},\frac{p_{ay_z} -\alpha^j_{ay_z}}{cy_{obs}}), \\
+& \text{Dirichlet}(\alpha^{j}_{s_x,n_y,ay_z}, \dots
+, \alpha^{j''}_{s_{x''},n_{y''},ay_{z''}}))\bigg]_{s_x,n_y,ay_z}
+  \end{split}
+\end{equation*}
+
+Interestingly, we can immediately see that the number of joiners to an
+entity is not directly related to that entity's population state,
+rather it's relation is to the projected population for an academic year
+and the non-send to entity transitions in the observed data.
+
+\subsubsection{Movers and Non Movers}
+
+
+The base form of the calculations should be by now familiar.
+
+\begin{equation*}
+  \begin{split}
+m^{partial}_{s_x,n_y,ay_z} =
+\text{Multinomial}( & \text{Binomial}(t_{s_x,n_y,ay_z} - l_{s_x,n_y,ay_z}, 
+ \text{Beta}(\alpha^m_{s_x,n_y,ay_z},\beta^m_{s_x,n_y,ay_z})),
+\\ &  \text{Dirichlet}(\alpha^{m'}_{s_x',n_y',ay_z'}, \dots,
+\alpha^{m''}_{s_{x''},n_{y''},ay_{z''}}))
+\end{split}
+\end{equation*}
+
+The first thing to note is that this a partial result - which we will
```

Comment:
what is the missing part? not sure what the "partial" superscript represents here